### ISSUE TITLE
Make Send Common Control Codes visible when in Terminal Mode

### ISSUE THREAD

**Issue by keithel** [microsoft/vscode-serial-monitor#173]
Type: Feature Request

TL;DR: Can Send Common Control Codes feature UI be moved from message send bar to top default toolbar or Additional Settings? 
(Or somewhere else more easily discoverable when you are in Terminal Mode)

---

While issue #22 is resolved, it took me a long time to figure out where this feature was, as when I connect to my device via serial port, I will immediately switch it into "Terminal Mode" via the "Toggle Terminal Mode" button in the top toolbar of the serial monitor.

Only when you turn off Terminal Mode, do you see the option to send common control codes.

![image](https://github.com/microsoft/vscode-serial-monitor/assets/156712/a6112d6b-e8d5-4fa0-8444-fbdc2b95b832)

Can this be moved from the bottom message send bar up into either the top default toolbar, or in Additional Settings? I think putting it in Additional Settings would be fine. It isn't visible all of the time, but it is more intuitive for users in Terminal mode to find than turning off terminal mode to find it.

![image](https://github.com/microsoft/vscode-serial-monitor/assets/156712/5c59cba1-7994-4eb3-a5ef-d437a30819da)

**Comment by gcampbell-msft** [comment]
@keithel Thank you for the suggestion, we will take it under consideration.

**Comment by keithel** [comment]
Of course, now I notice that you can just use the control codes with keys directly when you're in "Terminal Mode".. but I am guessing that won't work for *all* control key combinations.

**Comment by gcampbell-msft** [comment]
Revisiting this, as you pointed out, in Terminal Mode you can enter the control codes with keys directly when in Terminal Mode. Because of this, I'm going to close out this item. If we end up adding more buttons for control codes that aren't available through keys, then we will consider revisiting this. 

THanks! @keithel
